versioned_docs/version-v2.18.x/user-guide/api-mediation/configuration-client-certificates.md

```diff
@@ -3,9 +3,9 @@
 :::info Roles: system programmer, system administrator, security administrator
```

Comment:
Generally be careful with addition of the x509, while it is true that the certificates we support are x509 we also use the term in another way which may confuse people.

Reply:
The reason behind this addition is that there is a terminology discrepancy between references in Zowe Docs to just client certificates and references in ESZ that describe x.509 client certificates. It should be clear that the certificates are x509 in the Zowe doc if that is where we are sending ESZ customers.
![x509 config section in ESZ](https://github.com/user-attachments/assets/e4fc22d7-ce4a-4e4e-a7f2-3cf16577e63b)

Reply:
Ok, Then we should use X.509 as the exact name in the Zowe Docs as well.